BTrees/BTreeTemplate.c

```diff
@@ -219,6 +219,8 @@ BTree_check(BTree *self)
     return result;
 }
 
+#define _GET_KEY_ERROR 1
+#define _GET_ALL_ERROR 0
```

Comment:
In conjunction with the `ignore_key_err` parameter name below, these names seem a little weird to me:  could we come up with clearer ones?